website/docs/compute/deployment/cloud-deployment/configure-webhooks.md

```diff
@@ -42,8 +42,8 @@ webhooks:
        headers: *commonHeaders
 ```
 
-## Hook points
-### `builder.function.promoted`
+## Hook point
+ `builder.function.promoted`
```

Comment:
I know we mentioned that this seems weird being a list with only one thing in it, but now that I see this I feel like it's weirder. `Hook point` in particular feels very off.